### PR TITLE
Fix(es): outstanding feedback on course preview block

### PR DESCRIPTION
**Pull request by e-wai**
## Implementation Description
- Some outstanding changes building on https://github.com/uwblueprint/rowan-house/pull/95
  - [x] Add courseId, moduleId to specify route link for `ModulePreview` component
  - [x] Remove `ModulePreview` text highlight on hover
  - [x] Migrate dummy data out of specific components
  - [x] Extend horizontal line under courses / create new course title
  - [x] Moved `EditActionsKebabMenu` from "common" to "admin" since I didn't see it used elsewhere?

TBD:
- need to figure out how to make Tag sit at base of the ModulePreview box no matter length of title

## Screenshots
https://user-images.githubusercontent.com/40974372/158976783-c753f0fa-f063-4184-85b6-9a437af41ebd.mov

## What should reviewers focus on?
- make sure all changes make sense, suggestions for TBD item very welcome

## Testing Procedure
- What test(s) should we run to make sure your code works?
   - What ways could the input be wrong?
   - Is it secure? Could nefarious users access/break it?

Visual QA at the moment

## Things to Note
- Additional dependencies/libraries you've added?
- Things you'd want to know when coming back to the code after a few months

n/a
easily testable?

## Checklist
- [x] Ensure code follows style guide by running the linters
- [x] I have updated the documentation or deemed it unnecessary
- [ ] I have linked the relevant issue in this PR
- [x] I have requested a review from the PL, as well as other dev(s) (and designers if necessary)

### CODE REVIEW

File: frontend/src/types/AdminDashboardTypes.ts [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+  moduleId: string;
   title: string;
   published: boolean;
   imageLink: string;
 }
 
+export interface ModulePreviewProps extends Module {
+  courseId: string;
+}
+
```
Comment (e-wai) [inline frontend/src/types/AdminDashboardTypes.ts:11]: kept this separate since I don't expect `courseId` to exist on the GraphQL module response object?

**Review by Shrinjay — APPROVED** [review]
LGTM, handles my concerns anyways

**Review by draymond63 — APPROVED** [review]
Looks good to me!

**Comment by e-wai** [comment]
Will merge as is, minor change to be made in cleaning up the `ModulePreview` -> single link, changing the tag so that it sits lower on the card no matter length of title

**Pull request merged**